packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -327,6 +330,19 @@ export default function TranslationView() {
         onVerseChange={(verseId) =>
           navigate(`/interlinear/${language}/verses/${verseId}`)
         }
+        navigateToNextUnapprovedVerse={async () => {
```

Comment:
The only problem I can see with that we couldn't set `loadedFromNextButton.current` directly in the `TranslationToolbar`. A couple ways around this:
- Pass `loadedFromNextButton` to `TranslationToolbar` (since it's a reference).
- Add a flag parameter to `onVerseChange` to indicate whether the verse should be considered loaded from the next button.

I know this is a small decision, but what do you think we should do?

Reply:
Done.